idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/components/KtSymbolDeclarationRendererProvider.kt

```diff
@@ -63,6 +66,19 @@ data class KtDeclarationRendererOptions(
      * Approximate Kotlin not-denotable types into denotable for declarations return type
      */
     val approximateTypes: Boolean = false,
+
+    /**
+     * Declaration header is something like `abstract class`, `fun`, or `private interface` in a declaration.
+     */
+    val renderDeclarationHeader: Boolean = true,
+
+    /**
+     * Whether to forcefully add `override` modifier when rendering functions or properties. Note that the [modifiers] option still
+     * controls whether `override` is rendered. That is, if [modifiers] don't contain `override`, then this flag does not have any effect.
+     */
+    val forceOverrideModifier: Boolean = false,
```

Comment:
In what case will `modifiers` contain `override` but will _not_ be rendered if `forceOverrideModifier = false`?

Reply:
`forceOverrideModifer` equal false means don't alter the behavior. Just do what the fir declarations said. This flag is only to force *adding* `override` flag if it's not present.